src/OrderFormAcceptanceTests.Steps/Steps/PreviewOrderSummary.cs

```diff
@@ -1,344 +1,393 @@
+        {
+            SetOrderCatalogueSectionToComplete();
+            new AdditionalServices(Test, Context).GivenTheSupplierAddedToTheOrderHasAnAdditionalServiceWithAPatientFlatPrice();
+            var orderItem = new OrderItem().GenerateAdditionalServiceOrderItemWithVariablePricedPerPatient((Order)Context["CreatedOrder"]);
+            orderItem.Create(Test.ConnectionString);
+            Context.Add("CreatedOrderItem", orderItem);
+        }
+
+        [Given(@"there are one or more Order items summarised in the Order items \(recurring cost\) table")]
+        public void GivenThereAreOneOrMoreOrderItemsSummarisedInTheOrderItemsRecurringCostTable()
+        {
+            var onDemandOrderItem = new OrderItem().GenerateOrderItemWithFlatPricedVariableOnDemand((Order)Context["CreatedOrder"]);
+            onDemandOrderItem.Create(Test.ConnectionString);
+
+            var declarativeOrderItem = new OrderItem().GenerateOrderItemWithFlatPricedVariableDeclarative((Order)Context["CreatedOrder"]);
+            declarativeOrderItem.Create(Test.ConnectionString);
+
+            var patientOrderItem = new OrderItem().GenerateOrderItemWithFlatPricedVariablePerPatient((Order)Context["CreatedOrder"]);
+            patientOrderItem.Create(Test.ConnectionString);
+        }
+
+        [Then(@"the Total cost for one year is the result of the Total cost for one year calculation")]
+        public void ThenTheTotalCostForOneYearIsTheResultOfTheTotalCostForOneYearCalculation()
+        {
+            var actual = Test.Pages.OrderForm.GetTotalAnnualCost();
+
+            const string expectedTotalAnnualCost = "1,316,535.00";
+            actual.Should().Be(expectedTotalAnnualCost);
+        }
+
+        [Then(@"the Total cost for one year is expressed as two decimal places")]
+        public void ThenItIsExpressedAsTwoDecimalPlaces()
+        {
+            const int decimalPointPartIndex = 1;
+            const int expectedDecimalPointLength = 2;
+
+            var actual = Test.Pages.OrderForm.GetTotalAnnualCost().Split('.')[decimalPointPartIndex].Length;
+
+            actual.Should().Be(expectedDecimalPointLength);
+        }
+
+        [Then(@"the Total monthly cost is the result of the Total monthly cost calculation")]
+        public void ThenTheTotalMonthlyCostIsTheResultOfTheTotalMonthlyCostCalculation()
+        {
+            var actual = Test.Pages.OrderForm.GetTotalMonthlyCost();
+
+            const string expectedTotalAnnualCost = "109,711.25";
+            actual.Should().Be(expectedTotalAnnualCost);
+        }
+
+        [Then(@"the Total monthly cost is expressed as two decimal places")]
+        public void ThenTheTotalMonthlyCostIsExpressedAsTwoDecimalPlaces()
+        {
+            const int decimalPointPartIndex = 1;
+            const int expectedDecimalPointLength = 2;
+
+            var actual = Test.Pages.OrderForm.GetTotalMonthlyCost().Split('.')[decimalPointPartIndex].Length;

```

Comment:
whats the value in the var decimalPointPartIndex ? just to remove the magic number from this line?

Reply:
Yeah just using a constant here to hopefully provide a little context rather than specifying the magic number 1.